pkg/virt-operator/install-strategy/create.go

```diff
@@ -1415,6 +1418,66 @@ func addOrRemoveSSC(targetStrategy *InstallStrategy,
 	return nil
 }
 
+func syncPodDisruptionBudgetForDeployment(deployment *appsv1.Deployment, clientset kubecli.KubevirtClient, kv *v1.KubeVirt, expectations *util.Expectations, stores util.Stores) error {
+	podDisruptionBudget := components.NewPodDisruptionBudgetForDeployment(deployment)
+
+	imageTag := kv.Status.TargetKubeVirtVersion
+	imageRegistry := kv.Status.TargetKubeVirtRegistry
+
+	injectOperatorMetadata(kv, &podDisruptionBudget.ObjectMeta, imageTag, imageRegistry)
+
+	var cachedPodDisruptionBudget *policyv1beta1.PodDisruptionBudget
+
+	pdbClient := clientset.PolicyV1beta1().PodDisruptionBudgets(deployment.Namespace)
+
+	obj, exists, _ := stores.PodDisruptionBudgetCache.Get(podDisruptionBudget)
+
+	if exists {
+		cachedPodDisruptionBudget = obj.(*policyv1beta1.PodDisruptionBudget)
+		if objectMatchesVersion(&cachedPodDisruptionBudget.ObjectMeta, imageTag, imageRegistry) {
+			log.Log.V(4).Infof("poddisruptionbudget %v is up-to-date", cachedPodDisruptionBudget.GetName())
+			return nil
+		}
+		// Patch if old version
+		var ops []string
+
+		// Add Labels and Annotations Patches
+		labelAnnotationPatch, err := createLabelsAndAnnotationsPatch(&deployment.ObjectMeta)
+		if err != nil {
+			return err
+		}
+		ops = append(ops, labelAnnotationPatch...)
+
+		// Add Spec Patch
+		newSpec, err := json.Marshal(deployment.Spec)
```

Comment:
you are going to patch the deployment spec into the pdb?

Reply:
Eagle eyes, thanks!